examples/storehouse-steamcommunity.js

```diff
@@ -32,66 +34,67 @@ if (fs.existsSync('polldata.json')) {
 	manager.pollData = JSON.parse(fs.readFileSync('polldata.json'));
 }
 
-steam.login(logOnOptions, function(err, sessionID, cookies, steamguard) {
+steam.login(logOnOptions, function (err, sessionID, cookies, steamguard) {
 	if (err) {
-		console.log("Steam login fail: " + err.message);
+		console.log('Steam login fail: ' + err.message);
 		process.exit(1);
 	}
 
 	fs.writeFile('steamguard.txt', steamguard);
 
-	console.log("Logged into Steam");
+	console.log('Logged into Steam');
 
-	manager.setCookies(cookies, function(err) {
-		if (err) {
-			console.log(err);
+	manager.setCookies(cookies, function (cookiesErr) {
+		if (cookiesErr) {
+			console.log(cookiesErr);
 			process.exit(1); // Fatal error since we couldn't get our API key
 			return;
 		}
 
-		console.log("Got API key: " + manager.apiKey);
+		console.log('Got API key: ' + manager.apiKey);
 	});
 
-	steam.startConfirmationChecker(30000, "identitySecret"); // Checks and accepts confirmations every 30 seconds
+	steam.startConfirmationChecker(30000, 'identitySecret'); // Checks and accepts confirmations every 30 seconds
 });
 
-manager.on('newOffer', function(offer) {
-	console.log("New offer #" + offer.id + " from " + offer.partner.getSteam3RenderedID());
-	offer.accept(function(err) {
+manager.on('newOffer', function (offer) {
+	console.log('New offer #' + offer.id + ' from ' + offer.partner.getSteam3RenderedID());
+	offer.accept(function (err) {
 		if (err) {
-			console.log("Unable to accept offer: " + err.message);
+			console.log('Unable to accept offer: ' + err.message);
 		} else {
 			steam.checkConfirmations(); // Check for confirmations right after accepting the offer
-			console.log("Offer accepted");
+			console.log('Offer accepted');
 		}
 	});
 });
 
-manager.on('receivedOfferChanged', function(offer, oldState) {
-	console.log(`Offer #${offer.id} changed: ${TradeOfferManager.ETradeOfferState[oldState]} -> ${TradeOfferManager.ETradeOfferState[offer.state]}`);
+manager.on('receivedOfferChanged', function (offer, oldState) {
+	console.log(`Offer #${offer.id} changed: \
+${TradeOfferManager.ETradeOfferState[oldState]} -> ${TradeOfferManager.ETradeOfferState[offer.state]}`);
 
-	if (offer.state == TradeOfferManager.ETradeOfferState.Accepted) {
-		offer.getReceivedItems(function(err, items) {
+	if (offer.state === TradeOfferManager.ETradeOfferState.Accepted) {
```

Comment:
Should be, but I'm talking about the general case. I don't want to start worrying about === everywhere when == works just as well, as long as you know what you're doing. Not to mention the possible breakage if people are doing stuff that relies on internal usage of ==. Steam appids are a good example. I wouldn't be surprised at all if people are using them as both ints and strings.